dashboard/datacenter.py

```diff
@@ -147,13 +147,6 @@ async def get_node_info(cls, node_id, get_summary=False):
 
         # Merge GcsNodeInfo to node physical stats
         node_info["raylet"].update(node)
-        # Add "is_head_node" field
-        # TODO(aguo): Grab head node information from a source of truth
-        node_info["raylet"]["is_head_node"] = (
-            cls.head_node_ip == node_physical_stats.get("ip")
-            if node_physical_stats.get("ip")
-            else False
-        )
```

Comment:
have you checked that this is not used anywhere? I don't know if this is safe to delete

Reply:
It's used but now `GcsNodeInfo` already contains the `is_head_node` field so we no longer need to set manually here?